house-snark/src/relations/blender/parser.rs

```diff
@@ -0,0 +1,22 @@
+use anyhow::Result;
+
+use super::types::{FrontendAccount, FrontendMerkleRoot, FrontendNote};
+use crate::relations::blender::note::note_from_bytes;
+
+pub fn parse_frontend_note(frontend_note: &str) -> Result<FrontendNote> {
+    Ok(note_from_bytes(frontend_note.as_bytes()))
+}
+
+pub fn parse_frontend_merkle_root(frontend_merkle_root: &str) -> Result<FrontendMerkleRoot> {
+    Ok(note_from_bytes(frontend_merkle_root.as_bytes()))
+}
+
+pub fn parse_frontend_account(frontend_account: &str) -> Result<FrontendAccount> {
+    Ok(frontend_account.as_bytes().try_into().unwrap())
+}
+
+pub fn parse_frontend_merkle_path_single(
+    frontend_merkle_path_single: &str,
+) -> Result<FrontendNote> {
```

Comment:
Just because it's also `[u64; 4]` , like `Note`, it doesn't mean we should use the same type alias, this is confusing. It's fine if we have a separate type alias nodes in Merkle path, even if it's also `[u64; 4]` internally.